group_vars/lib_jobs/production.yml

```diff
@@ -54,6 +54,7 @@ transaction_error_feed_recipients: '{{ vault_lib_jobs_prod_transaction_error_fee
 voucher_feed_recipients: '{{ vault_lib_jobs_prod_voucher_feed_recipients }}'
 # Git
 git_lab_host: 'gitlab-prod-vm.lib.princeton.edu'
+git_lab_staging_host: 'gitlab-staging-vm.lib.princeton.edu'
```

Comment:
if this value is the same for staging and prod, could we put it in common.yml instead?